add_docs.py

```diff
@@ -301,7 +301,7 @@ def process(collection, path):  # pylint: disable-msg=too-many-locals
                             except KeyError:
                                 pass  # This module takes no options
 
-                            module_rst_path = Path(path, "docs", doc["module"] + ".rst")
+                            module_rst_path = Path(path, "docs", doc["module"] + "_{0}".format(plugin_type) + ".rst")
```

Comment:
'module' string also getting appended to files

Reply:
Yes, that is intended. Each doc is appended by the plugin type it is. I don't see any harm that it may cause. Also, I feel it's much cleaner than having separate convention just for a small subset of plugin types.

Reply:
The rst skel also follows the same pattern https://github.com/ansible-network/collection_prep/blob/master/plugin.rst.j2#L6

Reply:
Nice catch here, I missed the overlapping/same name issue.